UnityProject/Assets/Scripts/Items/ItemAttributesV2.cs

```diff
@@ -91,6 +91,17 @@ public float TraumaDamageChance
 		[EnumFlag]
 		public TraumaticDamageTypes TraumaticDamageType;
 
+		[SerializeField,
+			Range(0, 100),
+			Tooltip("How likely a player is to block an attack if they are holding this item in their active hand, 0% for never.")]
+		private float blockChance = 0;
+
+		public float BlockChance
```

Comment:
It would be better to choose `MultiFloat` so blocking has interest sources.
`blackChance` is the base, while `BlockChance` manages all sources. That way you don't have to manage when the stats change on things like energy swords when they're on/off.